hypha/templates/includes/passwordless_login_button.html

```diff
@@ -4,5 +4,5 @@
     href="{% url 'users:passwordless_login_signup' %}{% if next %}?next={{next}}{% endif %}"
 >
     {% heroicon_mini "envelope" size=18 class="inline align-text-bottom me-1" aria_hidden=true %}
-    {% trans "Log in " %}{% if ENABLE_PUBLIC_SIGNUP %} {% trans "or" %} {% trans "Sign in" %}{% endif %} {% trans "via" %} {% trans "e-mail" %}
+    {% trans "Log in without password" %}
```

Comment:
I like this a lot better, should we indicate that this is where you would also go to signup though as that isn't given anywhere else?

Reply:
My thinking was that if you clicked your way to the password login form you are most likely not needing to create an account.

Reply:
fair enough, all good by me!